### ISSUE TITLE
Some custom components may lack ARIA roles or labels

### ISSUE THREAD

**Issue by CCMumma** [TexasDigitalLibrary/Vireo#2177]
This addresses Vireo VPAT issue 4.1.2 Name, Role, Value. Some custom components may lack ARIA roles or labels. See also potentially related Github issues [1705](https://github.com/TexasDigitalLibrary/Vireo/issues/1705) and [1670](https://github.com/TexasDigitalLibrary/Vireo/issues/1670).

See also https://www.w3.org/WAI/WCAG22/quickref/#name-role-value
